package-lock.json

```diff
@@ -156,9 +142,9 @@
       },
       "dependencies": {
         "@angular-devkit/core": {
-          "version": "10.2.0",
-          "resolved": "https://registry.npmjs.org/@angular-devkit/core/-/core-10.2.0.tgz",
-          "integrity": "sha512-XAszFhSF3mZw1VjoOsYGbArr5NJLcStjOvcCGjBPl1UBM2AKpuCQXHxI9XJGYKL3B93Vp5G58d8qkHvamT53OA==",
+          "version": "10.2.1",
```

Comment:
Is this and all the Angular changes correct? Doesn't seem like they should have updated for removing one module.

Reply:
Looks like what would've happened if one ran `npm update`